vars/durationMinutes.groovy

```diff
@@ -0,0 +1,24 @@
+// vars/durationMinutes.groovy
+
+  /**
+   * durationMinutes step method
+   *
+   * This does not create the file, it is so the methods
+   * that create / fetch / read the STAGE status file all use
+   * the same name and it is properly URLEncoded.
```

Comment:
Is this copy'n'pasted from somewhere?  It doesn't seem relevant to this function.